### DISCUSSION TITLE
How to disable all git interations?

### DISCUSSION THREAD

**Discussion by mdouchement** [gopasspw/gopass#2361] [Ideas]
I had to update `gopass` from a very old version and now the edition makes lots of assumptions that don't fit the repository I use.
Is it possible to disable (at least):
- git pull
- git push

**Comment by dominikschulz** [comment]
I don't think I can follow you.

What exactly is causing trouble and how would you like to to behave?

You can entirely disable git if you use the `fs` backend, i.e. if you remove the `.git` folder.
If that exists it will automatically use the `gitfs` backend.
